package.json

```
@@ -5,6 +5,7 @@
   "description": "CodeQL action",
   "scripts": {
```

Suggestion: add a `bin` property to `package.json`, which resolves to the new CLI being introduced. This will allow new users to run the CLI directly, e.g. via `npx ...`.

```suggestion
  "bin": {
    "codeql-cli": "cli/code-scanning-cli.js"
  },
  "scripts": {
```